tmt/base.py

```diff
@@ -3099,9 +3099,10 @@ def init(
             tree = None
 
         # Create a new tree
+        fmf_dir = path / '.fmf'
         if tree is None:
             if dry:
-                echo(f"Tree '{path}' would be initialized.")
+                echo(f"Would create '{fmf_dir}'.")
             else:
```

Comment:
Perhaps, what about using consistently `fmf tree root` or `metadata tree root` across all the messages?